src/style.css

```diff
@@ -0,0 +1,34 @@
+body {
+  margin: 20px;
+  background-color: #000;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.external {
+  background-color: #3a92c3;
+  width: 350px;
+}
+
+section,
+.external {
+  padding: 10px;
+  border: 1px solid  #e95d49;
+}
+
+section {
+  background-color: #fea90d;
+}
+
+.inner {
+  border: 1px solid #e95d49;
```

Comment:
you give this border to every layer, maybe give them a common class which will style the red border?

Reply:
i add a common class border with this style, check pls)